Cargo.toml

```diff
@@ -18,7 +21,16 @@ risc0-zkvm = { git = "https://github.com/risc0/risc0.git" }
 # Jolt
 jolt = { package = "jolt-sdk", git = "https://github.com/a16z/jolt", features = ["host"] }
 
+# Aligned SDK
+aligned-sdk = { git = "https://github.com/yetanotherco/aligned_layer" }
```

Comment:
lets pin the version of the sdk so that it doesn't break while it is unstable

Reply:
done